cli/azd/pkg/project/framework_service_dotnet_test.go

```diff
@@ -63,6 +63,12 @@ func TestBicepOutputsWithDoubleUnderscoresAreConverted(t *testing.T) {
 
 func Test_DotNetProject_Restore(t *testing.T) {
 	var runArgs exec.RunArgs
+	ostest.Chdir(t, t.TempDir())
```

Comment:
Could consider adding a happy-path test that verifies `dotnet` working with multiple project files, and the unique project file was specified in `createTestServiceConfig`